src/smc-util/sync/editor/generic/sync-doc.ts

```diff
@@ -260,7 +263,7 @@ export class SyncDoc extends EventEmitter {
     ]);
 
     if (this.change_throttle) {
-      this.emit_change = throttle(this.emit_change, this.change_throttle);
+      this.emit_change = throttle(this._emit_change, this.change_throttle);
```

Comment:
Don't do this.  The leading underscore convention is something used in Python because there is no actual notion of "private" in Python.  In coffeescript and javascript it is useful too, but in Typescript it is totally unnecessary.   I've been systematically removing use of leading underscores in our typescript code, since it isn't needed...